www/js/diary/list.js

```diff
@@ -76,12 +76,33 @@ angular.module('emission.main.diary.list',['ui-leaflet', 'nvd3ChartDirectives',
       return ($scope.dark_theme)? "ion-view-background-dark" : "ion-view-background"; 
     }
     $scope.datePickerClass = function() {
-
     }
     $scope.listCardClass = function() {
-      return ($scope.dark_theme)? "list card list-card-dark" : "list card list-card"; 
+      if ($window.screen.width <= 320) {
+        return ($scope.dark_theme)? "list card list-card-dark list-card-sm" : "list card list-card list-card-sm"; 
+      } else if ($window.screen.width <= 375) {
+        return ($scope.dark_theme)? "list card list-card-dark list-card-md" : "list card list-card list-card-md"; 
+      } else {
+        return ($scope.dark_theme)? "list card list-card-dark list-card-lg" : "list card list-card list-card-lg"; 
+      }
+      
+    }
+    $scope.listColLeftClass = function(margin) {
+      if (margin == 0) {
+        return ($scope.dark_theme)? "col-50 list-col-left-dark" : "col-50 list-col-left";   
+      } else {
+        return ($scope.dark_theme)? "col-50 list-col-left-margin-dark" : "col-50 list-col-left-margin"; 
+      }
+    }
+    $scope.listColRightClass = function() {
+      return ($scope.dark_theme)? "col-50 list-col-right-dark" : "col-50 list-col-right"; 
+    }
+    $scope.differentCommon = function(tripgj) {
```

Comment:
I'm not sure what you are trying to do there, and there are no screenshots, but I'm just going to merge now.